### DISCUSSION TITLE
Is it possible to have a tabbed layout of frames instead of windows?

### DISCUSSION THREAD

**Discussion by ArijanJ** [herbstluftwm/herbstluftwm#1554] [Q&A]
Is there any way to have a frame using the `max` layout hold other frames? I think it might be possible by using new desktops. 
If anyone has any idea how I would go about scripting this, please let me know. 

(Use case is having a browser on one tab, and two terminals side by side on the other tab, for example.)

Edit: I guess what I'm looking for is a frame that holds another frame. Maybe run a daemon that moves & resizes a fake monitor?

**Comment by RayZ0rr** [comment]
So basically you want grouping of windows? If its for terminal then tmux would be a good fit

**Comment by ArijanJ** [comment]
Closing this in a very timely manner as i3 suits my workflow more, but `tmux` or suckless' `tabbed` are good workarounds for this
